### PR TITLE
INC-487: Mock-up bare bones trends chart with meaningless data

### PR DESCRIPTION
**Pull request by ushkarev**
Proof of concept demonstration of how the styles could work. All data presented is random.

⚠️ This mockup uses SVG and CSS grid layout so won't work on "older" browsers. We're aiming for Firefox v60 and "modern" browsers so should be ok… probably?

## Incentive levels trends

trends chart

## Behaviour entry trends

trends chart

### CODE REVIEW

File: integration_tests/pages/analyticsBehaviourEntries.ts [hunk 1]
```diff
@@ -14,6 +14,10 @@ export default class AnalyticsBehaviourEntries extends AnalyticsPage {
     return cy.get('#table-prisoners-with-entries-by-location tbody tr')
   }
 
+  get entriesTrends(): PageElement<HTMLTableRowElement> {
+    return cy.get('#table-trends-entries tbody tr')
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply integration_tests/pages/analyticsBehaviourEntries.ts:18]: i had been planning to do that once the data being plotted isn't just random numbers… not really sure what's sensible/realistic to test at this point. the chart range thingy i did add tests for

[the remark this answers is not in the document]
Comment (ushkarev) [reply integration_tests/pages/analyticsBehaviourEntries.ts:18]: also, this page definition class is becoming really gross. i want to restructure it to be more systematic but haven't thought of a nice way yet

File: server/views/partials/analytics/trends.njk [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+  <table class="app-chart-table app-chart-table--trends" id="table-{{ graphId }}">
+    <caption class="govuk-visually-hidden">
+      {{ chartTitle }}
+    </caption>
+    <thead>
+      <tr aria-hidden="true" class="app-chart-trends-svg-layer">
+        <td>
+          <span>
+            {% if trends.plotPercentage %}
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply server/views/partials/analytics/trends.njk:82]: 👌 can also change the `if trends.plotPercentage` on line 129 to match too

File: assets/sass/components/_chart-table-trends.scss [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    th {
+      font-weight: normal;
+    }
+
+    td {
+      position: relative;
+      padding: govuk-spacing(2) 0;
+
+      &:first-child {
+        padding: 0 govuk-spacing(2) 0 0;
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply assets/sass/components/_chart-table-trends.scss:12]: ok, i've added a bunch of comments throughout the chart SCSS

**Review by xoen — DISMISSED** [review]

File: assets/sass/components/_chart-table-trends.scss [hunk 4]
```diff
@@ -0,0 +1,97 @@
+.app-chart-table--trends {
+  thead {
+    th {
+      font-weight: normal;
+    }
+
+    td {
+      position: relative;
+      padding: govuk-spacing(2) 0;
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply assets/sass/components/_chart-table-trends.scss]: i've pulled that out as `$chart-cell-pading` as it appears all over the place

File: integration_tests/pages/analyticsBehaviourEntries.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (neilmendum) [reply integration_tests/pages/analyticsBehaviourEntries.ts:18]: Sure we can come back to the Cypress tests once we have a json file with source data

**Review by neilmendum — APPROVED** [review]
LGTM

**Pull request merged**